docs/guides/automation/templates-automation-packer-vsphere.md

```diff
@@ -323,7 +323,7 @@ Then you have to provide the complete command to be entered during the installat
 
     This example takes the most complex case: using a static IP. If you have a DHCP server available, the process will be much easier.
 
-This is the most amusing part of the procedure: I'm sure you'll go and admire the VMware console during the generation, just to see the automatic entry of the commands during the boot.
+This is the most amusing part of the procedure: i am sure you will go and admire the VMware console during the generation, just to see the automatic entry of the commands during the boot.
```

Comment:
Hey @SergeCroise If you are proposing this change the "i am" should be capitalized "I am". It looks like you are opposed to all contractions in technical writing, and while I'm not opposed to changing this (it probably does make it easier for translators), they are not *technically* errors. It might be good when you are making these suggestions, to just note that you are changing them to improve the ability of translators to translate or some other reason. 

Otherwise this all looks good. Again, I've got no problems with removing the contractions in favor of the actual wording. Fix this capitalization issue and I will merge this.  Thank you!

Reply:
you are right, 
in fact, this is an informal phrase.